src/index.js

```diff
@@ -40,12 +40,17 @@ const renderWithCarbon = ({ env, rootElement, model, theme, selectionsAPI, app,
 
 export default function supernova(env) {
   return {
+    // Define the properties of the Generic Object
+    // and the shape of the data you expect to consume.
+    // The properties and data are handled by
+    // Qlik's Associative Engine (QAE).
     qae: {
-      properties: {
-        initial: properties,
-      },
-      data: data(env),
+      properties: { initial: properties },
+      data: data(),
     },
+    // Define the extension definition.
+    ext: ext(env),
+    // Render the visual part.
```

Comment:
This is explaining how nebula works, not the table specifically. I'm all for documenting this, but could it fit in nebula somehow?

Reply:
nebula has those docs somewhere, I will delete here